portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,13 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Salvador's Portfolio</h1>
+      <p>This is Salvador's portfolio.</p>
+      <p>I'm currently a STEP intern at Google from The University of Texas at El Paso.</p>
+      <img src = "/Images/Me.jpg" class = "myPortrait" \>
```

Comment:
The slash is going the wrong way. It's suppose to be `/>`. You can use this to validate your HTML https://www.freeformatter.com/html-validator.html